### PR TITLE
Added launch profiles for the Utility project

### PR DESCRIPTION
**Pull request by penev92**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -45,5 +45,19 @@
...
 		},
+		{
+			"name": "Launch Utility",
+			"type": "coreclr",
+			"request": "launch",
+			"program": "${workspaceRoot}/bin/OpenRA.Utility.dll",
+			"windows": {
+				"program": "${workspaceRoot}/bin/OpenRA.Utility.exe",
+			},
+			"args": ["all", "--docs", "release-20210321"],
```
Comment (pchote) [inline .vscode/launch.json]: Hardcoding a release tag in these seems like a bad idea.

Comment (penev92) [reply .vscode/launch.json]: It was just example arguments. I'd be fine with switching them out for anything else. I just think having an example/placeholder is better than not having one.

Comment (abcdefg30) [reply .vscode/launch.json]: Why not the usual `{DEV_VERSION}` then?

Comment (penev92) [reply .vscode/launch.json]: Done.

**Comment by penev92** [comment]
I don't know if it would be of any use, but I added a VSCode task to run the utility after a prompt by @abcdefg30 on Discord.

**Review by abcdefg30 — APPROVED** [review]

**Review by atlimit8 — APPROVED** [review]
Works for me,

**Comment by atlimit8** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/6f248e4fa6d2210abd5bf9ea839e79b779996450)

File: .vscode/launch.json [hunk 2]
```diff
@@ -45,5 +45,19 @@
 			"args": ["Game.Mod=ts", "Engine.EngineDir=.."],
 			"preLaunchTask": "build",
 		},
+		{
+			"name": "Launch Utility",
```
Comment (Mailaender) [inline .vscode/launch.json:49]: This is actually "Generate Trait Documentation",

File: .vscode/launch.json [hunk 3]
```diff
@@ -45,5 +45,19 @@
...
+			"type": "coreclr",
+			"request": "launch",
+			"program": "${workspaceRoot}/bin/OpenRA.Utility.dll",
+			"windows": {
+				"program": "${workspaceRoot}/bin/OpenRA.Utility.exe",
+			},
+			"args": ["all", "--docs", "{DEV_VERSION}"],
```
Comment (Mailaender) [inline .vscode/launch.json:56]: Why did you add this hard-coded when there is a more flexible task below?

File: OpenRA.Utility/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+  "profiles": {
+    "OpenRA.Utility": {
+      "commandName": "Project",
+      "commandLineArgs": "all --docs {DEV_VERSION}",
```
Comment (Mailaender) [inline OpenRA.Utility/Properties/launchSettings.json:5]: This seems redundant to the one beforehand.

**Comment by penev92** [comment]
@Mailaender the VS launch profile (`OpenRA.Utility/Properties/launchSettings.json`) and the VSCode launch configuration (`.vscode/launch.json`) are two entirely separate things and each is used by the respective IDE, so we need both.
The default values are there as a placeholder and an example on how to use them.
